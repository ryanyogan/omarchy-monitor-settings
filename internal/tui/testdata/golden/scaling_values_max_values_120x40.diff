Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                                                [1;34mDisplay Settings[0m                                                [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [35m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [1;35m🔧 Manual Scaling Controls[0m                                                                                    [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [33m╭──────────────────────────────────────────────────────────╮[0m                                                  [35m│[0m    
  [35m│[0m  [33m│[0m[40m                                                          [0m[33m│[0m                                                  [35m│[0m    
  [35m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)[0m[40m  [0m[33m│[0m                                                  [35m│[0m    
  [35m│[0m  [33m│[0m[40m                                                          [0m[33m│[0m                                                  [35m│[0m    
  [35m│[0m  [33m╰──────────────────────────────────────────────────────────╯[0m                                                  [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [1;34m⚙️ Scaling Controls[0m                                                                                           [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [40m [0m[1;34;40m1. Monitor Scale (Compositor-level)[0m[40m [0m                                                                         [35m│[0m    
  [35m│[0m  [40m [0m[32;40m   Current: 4.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x, 2.0x, 2.25x, 2.5x, 3.0x)[0m[40m [0m                 [35m│[0m    
  [35m│[0m  [40m [0m[37;40m   Scales everything immediately. Works with all apps.[0m[40m [0m                                                      [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [1;36m2. GTK Scale (Application-level)[0m                                                                              [35m│[0m    
  [35m│[0m  [32m   Current: 3x (Range: 1x - 3x, Integer only)[0m                                                                 [35m│[0m    
  [35m│[0m  [37m   Scales GTK apps (most Linux apps). Requires logout.[0m                                                        [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [1;33m3. Font DPI (Text rendering)[0m                                                                                  [35m│[0m    
  [35m│[0m  [32m   Current: 300 (Range: 72 - 288, Step: 12)[0m                                                                   [35m│[0m    
  [35m│[0m  [37m   Fine-grained text scaling. Works with most applications.[0m                                                   [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [1;32m📊 Preview Results[0m                                                                                            [35m│[0m    
  [35m│[0m    Effective Resolution: 480x270                                                                               [35m│[0m    
  [35m│[0m    Screen Real Estate: 25%                                                                                     [35m│[0m    
  [35m│[0m    Font DPI Multiplier: 3.1x                                                                                   [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m  [32m↑↓[0m[37m select control[0m  [36m←→[0m[37m adjust value[0m  [33m⏎[0m[37m apply all[0m  [35mesc[0m[37m back[0m                                                     [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m│[0m                                                                                                                [35m│[0m    
  [35m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                           [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                         [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                                                                                │    
  │  🔧 Manual Scaling Controls                                                                                    │    
  │                                                                                                                │    
  │  ╭──────────────────────────────────────────────────────────╮                                                  │    
  │  │                                                          │                                                  │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)  │                                                  │    
  │  │                                                          │                                                  │    
  │  ╰──────────────────────────────────────────────────────────╯                                                  │    
  │                                                                                                                │    
  │  ⚙️ Scaling Controls                                                                                           │    
  │                                                                                                                │    
  │   1. Monitor Scale (Compositor-level)                                                                          │    
  │      Current: 4.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x, 2.0x, 2.25x, 2.5x, 3.0x)                  │    
  │      Scales everything immediately. Works with all apps.                                                       │    
  │                                                                                                                │    
  │  2. GTK Scale (Application-level)                                                                              │    
  │     Current: 3x (Range: 1x - 3x, Integer only)                                                                 │    
  │     Scales GTK apps (most Linux apps). Requires logout.                                                        │    
  │                                                                                                                │    
  │  3. Font DPI (Text rendering)                                                                                  │    
  │     Current: 300 (Range: 72 - 288, Step: 12)                                                                   │    
  │     Fine-grained text scaling. Works with most applications.                                                   │    
  │                                                                                                                │    
  │  📊 Preview Results                                                                                            │    
  │    Effective Resolution: 480x270                                                                               │    
  │    Screen Real Estate: 25%                                                                                     │    
  │    Font DPI Multiplier: 3.1x                                                                                   │    
  │                                                                                                                │    
  │  ↑↓ select control  ←→ adjust value  ⏎ apply all  esc back                                                     │    
  │                                                                                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 8582
Actual length: 7361
Expected hash: 95fccf5130c418038429827eaf9738c7f3ab6f6929c84c606da7dbc0ff48f488
Actual hash: 160833373ef5d27d0659bbbbfaecea21773be11b857235da36510e25e4f4baf9
