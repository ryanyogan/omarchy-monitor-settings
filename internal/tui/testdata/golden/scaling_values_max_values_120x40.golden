# Visual Golden File
# Name: scaling_values_max_values
# Dimensions: 120x40
# Hash: 160833373ef5d27d0659bbbbfaecea21773be11b857235da36510e25e4f4baf9

  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                                                                                │    
  │  🔧 Manual Scaling Controls                                                                                    │    
  │                                                                                                                │    
  │  ╭──────────────────────────────────────────────────────────╮                                                  │    
  │  │                                                          │                                                  │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)  │                                                  │    
  │  │                                                          │                                                  │    
  │  ╰──────────────────────────────────────────────────────────╯                                                  │    
  │                                                                                                                │    
  │  ⚙️ Scaling Controls                                                                                           │    
  │                                                                                                                │    
  │   1. Monitor Scale (Compositor-level)                                                                          │    
  │      Current: 4.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x, 2.0x, 2.25x, 2.5x, 3.0x)                  │    
  │      Scales everything immediately. Works with all apps.                                                       │    
  │                                                                                                                │    
  │  2. GTK Scale (Application-level)                                                                              │    
  │     Current: 3x (Range: 1x - 3x, Integer only)                                                                 │    
  │     Scales GTK apps (most Linux apps). Requires logout.                                                        │    
  │                                                                                                                │    
  │  3. Font DPI (Text rendering)                                                                                  │    
  │     Current: 300 (Range: 72 - 288, Step: 12)                                                                   │    
  │     Fine-grained text scaling. Works with most applications.                                                   │    
  │                                                                                                                │    
  │  📊 Preview Results                                                                                            │    
  │    Effective Resolution: 480x270                                                                               │    
  │    Screen Real Estate: 25%                                                                                     │    
  │    Font DPI Multiplier: 3.1x                                                                                   │    
  │                                                                                                                │    
  │  ↑↓ select control  ←→ adjust value  ⏎ apply all  esc back                                                     │    
  │                                                                                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    